includes/class-wc-stripe-helper.php

```
@@ -277,6 +277,31 @@ public static function get_stripe_amount( $total, $currency = '' ) {
+		if ( in_array( $currency, self::no_decimal_currencies(), true ) ) {
+			$amount   = (float) absint( $stripe_amount );
+			$decimals = 0;
+		} elseif ( in_array( $currency, self::three_decimal_currencies(), true ) ) {
+			$amount   = (float) $stripe_amount / 1000;
+			$decimals = 3;
+		} else {
+			$amount   = (float) $stripe_amount / 100;
+			$decimals = 2;
+		}
+		return wc_format_decimal( $amount, $decimals );
+	}
```

PHPMD (2.15.0)

[error] The method get_woocommerce_amount_from_stripe_amount is not named in camelCase. (undefined)

(CamelCaseMethodName)